datasets/totto/totto.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import json
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+ToTTo is an open-domain English table-to-text dataset with over 120,000 training examples that proposes a controlled generation task: given a Wikipedia table and a set of highlighted table cells, produce a one-sentence description.
+"""
+_HOMEPAGE_URL = ""
+_URL = "https://storage.googleapis.com/totto/totto_data.zip"
+_CITATION = """\
+@inproceedings{parikh2020totto,
+  title={{ToTTo}: A Controlled Table-To-Text Generation Dataset},
+  author={Parikh, Ankur P and Wang, Xuezhi and Gehrmann, Sebastian and Faruqui, Manaal and Dhingra, Bhuwan and Yang, Diyi and Das, Dipanjan},
+  booktitle={Proceedings of EMNLP},
+  year={2020}
+ }
+"""
+
+
+class ToTTo(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Totto(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `totto`